src/Provider/HostIp/HostIpXml.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Geocoder package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Geocoder\Provider\HostIp;
+
+use Geocoder\Collection;
+use Geocoder\Model\AddressCollection;
+
+/**
+ * @author Oleg Andreyev <oleg@andreyev.lv>
+ */
+final class HostIpXml extends AbstractHostIp
+{
+    /**
+     * @var string
+     */
+    const ENDPOINT_URL = 'http://api.hostip.info/get_xml.php?ip=%s&position=true';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName(): string
+    {
+        return 'host_ip_xml';
+    }
+
+    /**
+     * @param string $url
+     *
+     * @return Collection
+     */
+    protected function executeQuery(string $url): AddressCollection
+    {
+        $content = $this->getUrlContents($url);
+        $xml = \simplexml_load_string($content);
```

Comment:
The `\` is not necessary.

```suggestion
        $xml = simplexml_load_string($content);
```

Reply:
replaced with `use`